src/hooks/posters.ts

```diff
@@ -71,24 +72,55 @@ export const usePosters = (posterHallId: string) => {
   const fuseVenues = useMemo(
     () =>
       new Fuse(filteredPosterVenues, {
-        keys: ["poster.title", "poster.authorName", "poster.categories"],
+        keys: [
+          "name",
+          "poster.title",
+          "poster.authorName",
+          "poster.categories",
+        ],
+        threshold: 0.2, // 0.1 seems to be exact, default 0.6: brings too distant if anyhow related hits
+        ignoreLocation: true, // default False: True - to search ignoring location of the words.
+        findAllMatches: true,
```

Comment:
[note] See the following for more details on these options + how the scoring works:

- https://fusejs.io/api/options.html
- https://fusejs.io/concepts/scoring-theory.html#scoring-theory

Reply:
See also my suggestion elsewhere in this review of using extended search instead of trying to hack our own tokenisation/support of exact matching quoted terms.

- https://fusejs.io/api/options.html#useextendedsearch

Reply:
The conclusion on that other thread is likely that tokenisation is actually the best solution we're going to get, and that it probably doesn't make sense to add the extended search here.

Based on that, going to mark this as resolved.